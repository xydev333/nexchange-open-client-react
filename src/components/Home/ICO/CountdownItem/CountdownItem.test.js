import React from 'react';
import { shallow } from 'enzyme';
import CountdownItem from './CountdownItem';

describe('CountdownItem', () => {
  it('renders correctly', () => {
    expect(shallow(<CountdownItem period="days" count="1" />)).toMatchSnapshot();
  });
});
