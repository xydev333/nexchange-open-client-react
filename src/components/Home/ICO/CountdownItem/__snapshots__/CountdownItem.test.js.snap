// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountdownItem renders correctly 1`] = `ShallowWrapper {}`;
